latex_fill_all.py

```diff
@@ -494,8 +494,14 @@ def insert_at_end(self, view, edit, value):
         '''
         if value:
             new_regions = []
+            cmd_mode = view.settings().get('command_mode', False)
             for sel in view.sel():
-                view.insert(edit, sel.end(), value)
+               # Workaround for Issue 1329 (based on @dhelonious)
+                if(cmd_mode and view.substr(sel.end()+1).strip() in ('', '\x00')
+                   and not view.substr(sel.end()) in '.|,|;|?|!|\'|"|(|)|[|]|{|}'.split('|')):
```

Comment:
Rather than

```
and not view.substr(sel.end()) in '.|,|;|?|!|\'|"|(|)|[|]|{|}'.split('|')):
```

You can do 

```
and not view.substr(sel.end()) in tuple('.,;?!\'"()[]{}'):
```